### PR TITLE
Compute svd

### PR DESCRIPTION
**Pull request by mhansen1**
ComputeSVD module. Test network is in the base directory.

### CODE REVIEW

**Comment by dcwhite** [comment]
@mhansen1 Bad merge--you didn't resolve conflicts. Remember to build after merging to make sure the merge is good.

**Comment by mhansen1** [comment]
Ya, I'm building it now to make sure that there are no errors.

**Comment by mhansen1** [comment]
Should be good now. Successfully built the program after the merge and tested it out.

File: src/Core/Algorithms/Math/ComputeSVD.cc [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+using namespace SCIRun;
+using namespace SCIRun::Core::Algorithms;
+using namespace SCIRun::Core::Datatypes;
+using namespace SCIRun::Core::Algorithms::Math;
+
+void ComputeSVDAlgo::run(MatrixHandle input_matrix, DenseMatrixHandle& LeftSingMat, DenseMatrixHandle& SingVals, DenseMatrixHandle& RightSingMat) const
+{
+	if(matrix_is::sparse(input_matrix))
+		matrix_convert::to_dense(input_matrix);
```
Comment (dcwhite) [inline src/Core/Algorithms/Math/ComputeSVD.cc]: @mhansen1 Careful--this function returns the converted matrix, without assigning it somewhere it won't change the argument. I'll commit a fix

**Pull request merged**